orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1387,6 +1387,37 @@ void RegisterTrainingOpSchemas() {
         }
       });
 
+  /**
+   * ClipGradNormInplace operator, taking multiple gradients as inputs (seq<tensor>).
```

Comment:
This can be set as the doc for the op using .SetDoc() I notice we dont do this anywhere in this file. Is this intentional? If not then this can be the first op to start this practice.